patch/kconfig-inclusions

```diff
@@ -126,6 +126,56 @@ CONFIG_SENSORS_EMC2305=m
 [marvell-arm64]
 CONFIG_MARVELL_AC5_WATCHDOG=m
 
+[pensando]
+CONFIG_ARCH_PENSANDO=y
+CONFIG_ARCH_PENSANDO_ELBA_SOC=y
+CONFIG_PENSANDO_SOC_CAPMEM_HUGEPAGE=y
+CONFIG_PENSANDO_SOC_PENFW=y
+CONFIG_EDAC_ELBA=y
+CONFIG_IRQ_PENSANDO=y
+CONFIG_MFD_PENSANDO_ELBASR=y
+CONFIG_RESET_ELBASR=y
+CONFIG_UIO_PCIEMAC=m
+CONFIG_UIO_PENMSI1=m
+CONFIG_UIO_PENGIC=m
+CONFIG_UIO_PENMSI=m
+CONFIG_ARM64_VA_BITS_39=y
+CONFIG_COMMON_CLK_CS2000_CP=y
+CONFIG_CONTEXT_TRACKING=y
+CONFIG_CRYPTO_MANAGER_DISABLE_TESTS=y
+CONFIG_DEVTMPFS_MOUNT=y
+CONFIG_EMBEDDED=y
+CONFIG_GENERIC_IRQ_DEBUGFS=y
+CONFIG_GPIO_DWAPB=y
+CONFIG_I2C_RD1173=y
+CONFIG_IP_PNP=y
+CONFIG_IP_PNP_BOOTP=y
+CONFIG_IP_PNP_DHCP=y
+CONFIG_IRQSOFF_TRACER=y
+CONFIG_KALLSYMS_ALL=y
+CONFIG_MEMTEST=y
+CONFIG_MMC_DEBUG=y
+CONFIG_MMC_SDHCI_CADENCE=m
+CONFIG_NO_HZ_FULL=y
+CONFIG_PENSANDO_SOC_CAPMEM_HUGEPAGE=y
+CONFIG_PENSANDO_SOC_PENFW=y
+CONFIG_PMBUS=y
```

Comment:
why is this y, why cannot we use module?

Reply:
changed to module